mass_editing/hooks.py

```diff
@@ -7,6 +6,5 @@ def uninstall_hook(cr, registry):
     cr.execute("""SELECT id FROM ir_act_window
                WHERE res_model = 'mass.editing.wizard'""")
     for res in cr.dictfetchall():
-        value = 'ir.actions.act_window,%s' % res.get('id')
-        cr.execute("DELETE FROM ir_values WHERE value = %s", (value, ))
+        cr.execute("DELETE FROM ir_act_window WHERE id = '%s'" % res.get('id'))
```

Comment:
1. Is this needed? Doesn't Odoo by itself automatically remove tables when an addon is uninstalled?
   - Yes: Add a docstring on why.
   - No: remove it.
2. This query is a little bit.... well, not KISS. Just `DELETE FROM ir_act_window WHERE res_model = 'mass.editing.wizard'` and you're done